<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form1</class>
 <widget class="QWidget" name="form1">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>120</horstretch>
    <verstretch>200</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>390</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TimeOfStudy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="filesMidia.qrc">
    <normaloff>:/imagens/fundo.png</normaloff>:/imagens/fundo.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fff;</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imagens/icon3.ico);
background-repeat:no-repeat;
background-position:cover;
border:1px solid rgb(94, 92, 100);
color: rgb(94,92,100);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(94, 92, 100);</string>
   </property>
   <property name="text">
    <string>Unidade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(94, 92, 100);</string>
   </property>
   <property name="text">
    <string>Em:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(94, 92, 100);</string>
      </property>
      <property name="text">
       <string>sim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(94, 92, 100);</string>
      </property>
      <property name="text">
       <string>não</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(94, 92, 100);</string>
      </property>
      <property name="text">
       <string>Lembrete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="cMsg">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(94, 92, 100);
color:rgb(94, 92, 100);</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:11pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>31</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(94, 92, 100);
color: rgb(94, 92, 100);</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="placeholderText">
       <string>00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(94, 92, 100);
color: rgb(94, 92, 100);
</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>minutos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>horas</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_mp3">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>289</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imagens/icon7.ico);
background-repeat:no-repeat;

border:1px solid rgb(94, 92, 100);</string>
   </property>
   <property name="text">
    <string>.mp3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(94, 92, 100);</string>
   </property>
   <property name="text">
    <string>Escolher música</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="filesMidia.qrc"/>
 </resources>
 <connections/>
</ui>
